From c9a6701cb917efcdbecc96c92c48402b3fe736e5 Mon Sep 17 00:00:00 2001
From: Lasse Laukkanen <ext-lasse.2.laukkanen@nokia.com>
Date: Mon, 29 Jun 2009 15:23:25 +0300
Subject: [PATCH] camerabin: don't reset capsfilters or zoom if error occurs

If an error occurs, application should set pipeline to NULL, and updating
zoom can actually block message handling if video device driver has failed
and video src element supports zooming using photography interface and S_CROP.
---
 gst/camerabin/gstcamerabin.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/gst/camerabin/gstcamerabin.c b/gst/camerabin/gstcamerabin.c
index 3b5ab63..42d470d 100644
--- a/gst/camerabin/gstcamerabin.c
+++ b/gst/camerabin/gstcamerabin.c
@@ -3110,7 +3110,6 @@ gst_camerabin_handle_message_func (GstBin * bin, GstMessage * msg)
           GST_MESSAGE_SRC (msg));
       g_mutex_lock (camera->capture_mutex);
       if (camera->capturing) {
-        gst_camerabin_finish_image_capture (camera);
         camera->capturing = FALSE;
         g_cond_signal (camera->cond);
       }
